app/src/main/java/com/odysee/app/MainActivity.java

```diff
@@ -40,15 +40,7 @@
 import android.text.style.TypefaceSpan;
 import android.util.Base64;
 import android.util.Log;
-import android.view.Gravity;
-import android.view.KeyEvent;
-import android.view.LayoutInflater;
-import android.view.MenuItem;
-import android.view.View;
-import android.view.Menu;
-import android.view.ViewGroup;
-import android.view.ViewTreeObserver;
-import android.view.WindowManager;
+import android.view.*;
```

Comment:
I strongly prefer importing individual classes instead of using the wildcard import, as the latter clutters the namespace.

Reply:
I also do prefer it. This change was added by the IDE. I will undo it.